README.md

```diff
@@ -305,7 +306,17 @@ $yourEvent->source = [
 
 ### Limitations
 
-The Google Calendar API provides many options. This package doesn't support all of them. For instance, recurring events cannot be managed properly with this package. If you stick to creating events with a name and a date you should be fine.
+The Google Calendar API provides many options. This package doesn't support all of them. For instance, recurring events cannot be managed properly with this package. If you stick to creating events with a name and a date you should be fine. You can manage your recurring events using another package. This is implemented native by Google however you'll need the package RRule\RRule to parse it.
+```
+        $event = Event::find('EVENT_ID_WITH_RECURRING','CALENDAR_ID');
```

Comment:
Please use the same indentation as the rest of the readme, and mark this as a PHP code snippet.